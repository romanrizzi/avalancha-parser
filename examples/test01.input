check true         -- verdadero
check false        -- falso

-- Igualdad entre variables
check foo == bar

check
abcdefghijklmnopqrstuvwxyz_1234567890
==
a0987654321_zyxwvutsrqponmlkjihgfedcb

