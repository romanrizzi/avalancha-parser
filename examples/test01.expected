[
   "program",
   [],
   [
      [
         "check",
         [
            "true"
         ]
      ],
      [
         "check",
         [
            "false"
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "foo"
            ],
            [
               "var",
               "bar"
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "var",
               "abcdefghijklmnopqrstuvwxyz_1234567890"
            ],
            [
               "var",
               "a0987654321_zyxwvutsrqponmlkjihgfedcb"
            ]
         ]
      ]
   ]
]
